### DISCUSSION TITLE
Informações a serem guardadas no banco

### DISCUSSION THREAD

**Discussion by matheusmms031** [daniloraisi/go-ahead#4] [Ideas]
Acho que antes de tudo devemos saber quais informações serão guardadas no banco de dados, queria saber do dono do repositório(@daniloraisi) qual seria a estrutura de tabelas do projeto para assim ajudar na construção do mesmo...

**Comment by Yansb** [comment]
Podemos utilizar esse site para fazer o diagrama do banco de dados https://dbdiagram.io/d

**Reply by matheusmms031** [reply]
Sim também acho uma boa ideia porém antes mesmo de definir a estrutura do banco devemos saber quais são as features da aplicação.

No caso a gente tem que criar kkk

**Reply by matheusmms031** [reply]
Dai foi pensando nisso que criei um fork do repositório e tô escrevendo as features da aplicação, dai quando acabar eu faço um pull request, se liga só: [features](https://github.com/matheusmms031/go-ahead/blob/main/FEATURES.md)

**Reply by matheusmms031** [reply]
Criei alguma coisinha, se liga só: https://dbdiagram.io/d/Go-Ahead-65a6b57eac844320ae0c36b8
